src/reporter/reporter.py

```diff
@@ -104,7 +104,25 @@ def _validate_payload(payload):
             payload[attr].update({'value': None})
             log().warning(
                 'An entity update is missing value for attribute {}'.format(attr))
+  
 
+def _filter_empty_entities(payload):
+    log().info('Received payload: {}'.format(payload))
```

Comment:
isn't this redundant? there's already a log statement to output the whole entity array as we get it in the request:

https://github.com/smartsdk/ngsi-timeseries-api/pull/278/files#diff-f796da60daee45177c252e52fc78b4aaL120

Reply:
Hie @c0c0n3 , I have checked and removed log statement from def notify().  :)

Reply:
cool bananas!